src/feature/projects/components/ProjectSection.tsx

```diff
@@ -1,17 +1,12 @@
 import React from "react"
 import Box from "@mui/material/Box"
 import Stack from "@mui/material/Stack"
-import {
-  GatsbyImage,
-  getImage,
-  IGatsbyImageData,
-  ImageDataLike,
-} from "gatsby-plugin-image"
+import { IGatsbyImageData } from "gatsby-plugin-image"
 import SeoLink from "../../../components/shared/SeoLink"
 import { ProjectSectionProps } from "../types"
 import { ProjectCard } from "./ProjectCard"
 import { ProjectContent } from "./ProjectContent"
-import { ProjectImage } from "./ProjectImage"
+import { ItemImage } from "../../guide/GuideItem"
```

Comment:
Non sono un grande fan di importare component da altre feature.
Il fatto che il component sia dentro la folder di feature, indica che debba essere solo usato all'interno di quella specifica feature, così da dare al dev la garanzia che quello che stia modificando venga modificato solo e unicamente in quel determinato posto e non stia distruggendo un altro component. 

Quindi se si vede che si ha il bisogno di utilizzare un component all'interno di più feature esso va migrato nella folder di components generale, così da rendere esplicito che si sta modificando un componente comune.